services/121-service/test/helpers/registration.helper.ts

```diff
@@ -150,12 +150,13 @@ export async function waitForDeleteRegistrations({
     // Get payment transactions
     let totalRegistrationSuccesfullyDeleted = 0;
     for (const referenceId of referenceIds) {
-      const getEventResponse = await getEvents(
+      const getEventResponse = await getEvents({
```

Comment:
If we're nitpicking on "naming variables" anyway... Then lets also go for `getEventsResponse` :D